packages/components/htz-components/src/components/HtzLink/HtzLink.js

```diff
@@ -165,7 +169,18 @@ const HtzLink = React.forwardRef(
             }
             : ref
         }
-        onClick={passedOnClick}
+        onClick={
+          typeof passedOnClick === 'function'
+            ? e => {
+              // if no passedOnClick function passed, redirect using href.
+              e.preventDefault();
+              passedOnClick && passedOnClick();
```

Comment:
`passedOnClick` should be forwarded the event. We don't know anything about it, and it's possible that it is using it internally:
```suggestion
              passedOnClick && passedOnClick(e);
```